src/asserts/aba-account-number-assert.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+
+const { Validator, Violation } = require('validator.js');
+
+/**
+ * Account number regex.
+ */
+
+const accountNumberRegex = /^\d{8,17}$/;
```

Comment:
Also, we don't seem to be validating the digit-only part. Both ach gateway and backend seem to allow letters, are we sure about this one?

Reply:
I'll double check.

Reply:
Based on the information I've read on Plaid's website, I've changed updated the regexp range from 5 to 17 characters where each character can be:
- a digit
- a lowercase letter
- an uppercase letter